### PR TITLE
Wffhcohort-439

### PR DESCRIPTION
**Pull request by shelleyMurphy**
https://jira.wh-sdlc.watson-health.ibm.com/browse/WFFHCOHORT-439

Moving the translation classes to their own module, with a translation entry point in the cli module.

### CODE REVIEW

**Review by dkwasny-ibm — CHANGES_REQUESTED** [review]
Some minor suggestions

File: cohort-cli/pom.xml [hunk 1]
```diff
@@ -140,7 +146,13 @@
 			<artifactId>wiremock-jre8</artifactId>
 			<scope>test</scope>
 		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>com.ibm.cohort</groupId>
+            <artifactId>cohort-translation</artifactId>
+            <version>0.0.1-SNAPSHOT</version>
```
[the remark this answers is not in the document]
Comment (shelleyMurphy) [reply cohort-cli/pom.xml:152]: my for-no-reason-at-all panicked rebase squashed this change, I'll recommit it along with the rest of your suggestions

File: cohort-translation/src/test/resources/model-info.xml [hunk 2]
```diff
@@ -0,0 +1,624 @@
...
+<!--
+  ~ /*
+  ~  * (C) Copyright IBM Corp. 2021
+  ~  *
+  ~  * SPDX-License-Identifier: Apache-2.0
+  ~  */
+  -->
+
+<modelInfo xmlns="urn:hl7-org:elm-modelinfo:r1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" name="HI" version="0.0.1" url="http://mock.hi.model" targetQualifier="IDontKnow" patientClassName="DrugClaim" patientBirthDatePropertyName="none">
+    <typeInfo namespace="HI" name="DrugClaim" identifier="http://mock.hi.model/DrugClaim" label="DrugClaim" retrievable="true" xsi:type="ClassInfo">
```
Comment (msargentibm) [inline cohort-translation/src/test/resources/model-info.xml:11]: Given this is HI-based, are we allowed to have it in the public repo? Might be a question for a morning meeting/Slack.

Comment (shelleyMurphy) [reply cohort-translation/src/test/resources/model-info.xml:11]: It's a fair question, I think you're right. For now I'm trading it out for an already committed model info.

File: cohort-cli/pom.xml [hunk 3]
```diff
@@ -140,7 +146,13 @@
...
+            <scope>compile</scope>
+        </dependency>
+    </dependencies>
```
[the remark this answers is not in the document]
Comment (shelleyMurphy) [reply cohort-cli/pom.xml:156]: I should have resolved all of your comments now

**Review by dkwasny-ibm — APPROVED** [review]
Looks good

**Review by msargentibm — APPROVED** [review]

**Pull request merged**